src/model/db/FileUpload.model.ts

```diff
@@ -87,6 +88,20 @@ export class FileUploadModel extends AbstractModel {
     })
     public encrypted: boolean;
 
+    @Column({
+        nullable: true,
+    })
+    public bucketToken: string | null;
+
+    @ManyToOne("BucketModel", "submissions", {
```

Comment:
`submissions` is wrong, this should be `files`

Reply:
Updated in my local copy

Reply:
Pushed change